web/content/docs/benchmarks/heat-transport-bhe/3D_BHE_GW_advection.pandoc

```diff
@@ -0,0 +1,68 @@
++++
+author = "Boyan Meng, Chaofan Chen, Haibing Shao"
+date = "2019-11-07T11:52:00+01:00"
+title = "BHE with groundwater advection"
+weight = 123
+project = "Parabolic/T/3D_BHE_GW_advection/BHE_GW_advection.prj"
+
+[menu]
+  [menu.benchmarks]
+    parent = "heat-transport-bhe"
+
++++
+
+{{< data-link >}}
+
+## Problem description
+When groundwater flow is present, advective heat transport in the soil matrix has to be considered. Hence, the governing equation for advective and conductive heat transport in porous media can be expressed (in 2D form) as follows:
+$$
```

Comment:
Maybe it should be mentioned that the thermal conductivity is assumed to be isotropic?

Reply:
Yes.  [Edited.](https://github.com/ufz/ogs/pull/2711/commits/b46591161fff7b7a7f53e2daaa0ddf6c99555214)